theme/templates/resource-landing-page/coverage.html

```diff
@@ -1,50 +1,36 @@
-{% if not metadata_form %}
+{% if not resource_edit_mode %}
     {# resource view mode #}
     <div class="col-xs-12 content-block">
+        <h3>Resource Level Coverage</h3>
         <div class="row">
-            <div class="col-xs-12">
-                <h3>Coverage</h3>
-            </div>
-            <div class="col-xs-12 content-block">
-                <div class="row">
-                    <div class="col-xs-12 col-sm-4">
-                        {% if spatial_coverage %}
-                            <h4>Spatial:</h4>
-                            <table class="info-table">
-                                <tbody>
-                                {% if spatial_coverage.projection %}
-                                    <tr>
-                                        <td>
-                                            <i class="fa fa-globe" aria-hidden="true"></i>
-                                            <strong>&nbsp;Coordinate System/Geographic Projection:&nbsp;</strong>
-                                        </td>
-                                        <td>
-                                            <span>{{ spatial_coverage.projection }}</span>
-                                        </td>
-                                    </tr>
-                                {% endif %}
-                                {% if spatial_coverage.units %}
-                                    <tr>
-                                        <td>
-                                            <i class="fa fa-tachometer" aria-hidden="true"></i>
-                                            <strong>&nbsp;Coordinate Units:&nbsp;</strong>
-                                        </td>
-                                        <td>
-                                            <span>{{ spatial_coverage.units }}</span>
-                                        </td>
-                                    </tr>
-                                {% endif %}
-                                </tbody>
-                            </table>
+            <div class="col-xs-12 col-sm-4">
+                {% if spatial_coverage %}
+                    <h4>Spatial</h4>
+                    {% if spatial_coverage.projection %}
+                        <div>
+                            <i class="fa fa-globe" aria-hidden="true"></i>
+                            <small>&nbsp;Coordinate System/Geographic Projection:&nbsp;</small>
+                        </div>
+                        <div>{{ spatial_coverage.projection }}</div>
+                        <br>
+                    {% endif %}
+                    {% if spatial_coverage.units %}
+                        <div>
+                            <i class="fa fa-tachometer" aria-hidden="true"></i>
+                            <small>&nbsp;Coordinate Units:&nbsp;</small>
```

Comment:
We can use CSS to replace the non-breaking space. Non-content related stuff should be considered to move to CSS.

Reply:
Agree. Will change.